### PR TITLE
Set default lb-method when the lb-provider is OVN 

### PR DESCRIPTION
**Pull request by m-bull**

### CODE REVIEW

File: roles/azimuth_capi_operator/defaults/main.yml [hunk 1]
```diff
@@ -80,7 +80,11 @@ azimuth_capi_operator_capi_helm_worker_root_volume_availability_zone: "{{ azimut
 
 # Options for LoadBalancer services
 # https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/openstack-cloud-controller-manager/using-openstack-cloud-controller-manager.md#load-balancer
-azimuth_capi_operator_capi_helm_openstack_loadbalancer_method:
+azimuth_capi_operator_capi_helm_openstack_loadbalancer_method: >-
+  {{-
+    'SOURCE_IP_PORT' if azimuth_capi_operator_capi_helm_openstack_loadbalancer_provider == 'ovn'
```
Comment (mkjpryor) [inline roles/azimuth_capi_operator/defaults/main.yml:85]: IMHO, putting the condition on the next line, like other examples, improves readability

File: roles/azimuth_capi_operator/defaults/main.yml [hunk 2]
```diff
@@ -80,7 +80,11 @@ azimuth_capi_operator_capi_helm_worker_root_volume_availability_zone: "{{ azimut
...
+    else unset
```
Comment (mkjpryor) [inline roles/azimuth_capi_operator/defaults/main.yml:86]: Should this be explicitly ROUND_ROBIN?

**Review by mkjpryor — CHANGES_REQUESTED** [review]

File: roles/azimuth_capi_operator/defaults/main.yml (hunk 1 above)
Comment (m-bull) [reply roles/azimuth_capi_operator/defaults/main.yml]: Agreed, fixed

File: roles/azimuth_capi_operator/defaults/main.yml (hunk 2 above)
Comment (mkjpryor) [reply roles/azimuth_capi_operator/defaults/main.yml:87]: Sorry - I meant instead of `unset` in the non-OVN case.

**Review by mkjpryor — APPROVED** [review]
LGTM

**Pull request merged**